config/environments/production.rb

```diff
@@ -44,7 +44,7 @@
   # config.action_cable.allowed_request_origins = [ 'http://example.com', /http:\/\/example.*/ ]
 
   # Force all access to the app over SSL, use Strict-Transport-Security, and use secure cookies.
-  # config.force_ssl = true
+  config.force_ssl = true
```

Comment:
cái này để làm gì e nhỉ

Reply:
Dạ nó buộc trình duyệt chuyển hướng HTTP sang HTTPS và nó sẽ đặt cookie được đánh dấu là an toàn.